app/src/main/java/com/example/trabalhador/view/SalaryBonusFragment.kt

```diff
@@ -57,7 +59,7 @@ class SalaryBonusFragment : Fragment() {
         binding.textValueSecond.text = "R$ ${"%.2f".format(salaryResult.secondPayment)}"
     }
 
-    private fun validOk(): Boolean =(
+    private fun validOk(): Boolean = (
             binding.textInputEditTextSalary.text.toString() != "" &&
                     binding.textInputEditTextSalary.text.toString().toFloat() != 0f &&
```

Comment:
Voce chegou a testar colocar um numero negativo? Poderiamos mudar essa logica pra considerar apenas de for `>` do que zero.

Pra praticar, podemos criar uma extension function do tipo para validar isso:
`fun Number.isBiggerThanZero() = if (this > 0) true else false`